From: Gordon Farquharson <gordonfarquharson@gmail.com>
Subject: [arm] Disable SCSI_ACARD
Date: Mon, 12 Nov 2007 04:08:38 +0000

ERROR: "__bad_udelay" [drivers/scsi/atp870u.ko] undefined!
make[1]: *** [__modpost] Error 1

diff --git a/drivers/scsi/Kconfig b/drivers/scsi/Kconfig
index a7a0813..fa6a4fe 100644
--- a/drivers/scsi/Kconfig
+++ b/drivers/scsi/Kconfig
@@ -396,7 +396,7 @@ config SCSI_7000FASST
 
 config SCSI_ACARD
 	tristate "ACARD SCSI support"
-	depends on PCI && SCSI
+	depends on PCI && SCSI && (!ARM || BROKEN)
 	help
 	  This driver supports the ACARD SCSI host adapter.
 	  Support Chip <ATP870 ATP876 ATP880 ATP885>
